src/main/java/org/kuali/kfs/pdp/businessobject/PaymentGroup.java

```diff
@@ -50,70 +61,236 @@
+ *
+ * <pre>
+ * &lt;complexType&gt;
+ *   &lt;complexContent&gt;
+ *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType"&gt;
+ *       &lt;sequence&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}payee_name"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}payee_id" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}payee_own_cd" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}address1"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}address2" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}address3" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}address4" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}city" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}state" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}zip" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}country" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}campus_address_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}payment_date" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}attachment_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}immediate_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}special_handling_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}taxable_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}nonresident_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}combine_group_ind" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}bank_code" minOccurs="0"/&gt;
+ *         &lt;element ref="{http://www.kuali.org/kfs/pdp/payment}detail" maxOccurs="27"/&gt;
+ *       &lt;/sequence&gt;
+ *     &lt;/restriction&gt;
+ *   &lt;/complexContent&gt;
+ * &lt;/complexType&gt;
+ * </pre>
  */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {"payeeName",
+    "payeeIdObj",
+    "payeeOwnerCd",
+    // CU customization
+    "customerInstitutionNumber",
+    "line1Address",
+    "line2Address",
+    "line3Address",
+    "line4Address",
+    "city",
+    "state",
+    "zipCd",
+    "country",
+    "campusAddress",
+    "paymentDate",
+    "pymtAttachment",
+    "processImmediate",
+    "pymtSpecialHandling",
+    "taxablePayment",
+    "nonresidentPayment",
+    "combineGroups",
+    "bankCode",
+    "paymentDetails"})
 public class PaymentGroup extends PersistableBusinessObjectBase {
```

Comment:
Given the other potential changes that may be needed based on my earlier comments, would it make sense to have some kind of labeling for each customized area (or at least a broad summary near this class declaration)?